setup.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+import os.path
+import errno
+import urllib.request
+from setuptools import setup
+
+
+version = "0.1.0"
+with open("./python/lsst/daf/butler/version.py", "w") as f:
+    print(f"""
+__all__ = ("__version__", )
+__version__='{version}'""", file=f)
+
+# The purpose of this setup.py is to build enough of the system to
```

Comment:
Almost worth including it as a git submodule...

Reply:
but then I'd still have to write a new `__init__.py` that didn't try to import all the C++ routines. I imagine that these github.com gets should be fairly stable for actions that themselves run on github.com.